tests/aci-e2e/e2e-aci_test.go

```diff
@@ -465,19 +465,19 @@ func TestComposeUpUpdate(t *testing.T) {
 	_, groupID := setupTestResourceGroup(t, c)
 
 	const (
-		composeFile              = "../composefiles/aci-demo/aci_demo_port.yaml"
-		composeFileMultiplePorts = "../composefiles/aci-demo/aci_demo_multi_port.yaml"
-		composeProjectName       = "acidemo"
-		serverContainer          = composeProjectName + "_web"
-		wordsContainer           = composeProjectName + "_words"
-		dbContainer              = composeProjectName + "_db"
+		composeFileVolumes              = "../composefiles/aci-demo/aci_demo_port_volumes.yaml"
+		composeFileVolumesMultiplePorts = "../composefiles/aci-demo/aci_demo_multiport_volumes.yaml"
+		composeProjectName              = "acidemo"
+		serverContainer                 = composeProjectName + "_web"
+		wordsContainer                  = composeProjectName + "_words"
+		dbContainer                     = composeProjectName + "_db"
 	)
 
 	t.Run("compose up", func(t *testing.T) {
 		dnsLabelName := "nginx-" + groupID
 		fqdn := dnsLabelName + "." + location + ".azurecontainer.io"
 		// Name of Compose project is taken from current folder "acie2e"
-		c.RunDockerCmd("compose", "up", "-f", composeFile, "--domainname", dnsLabelName)
+		c.RunDockerCmd("compose", "up", "-f", composeFileVolumes, "--domainname", dnsLabelName)
```

Comment:
Only need a `volume create` before this (and the cleanup that goes with it ; volumes are not deleted with resource groups I think), like https://github.com/gtardif/compose-cli/blob/main/tests/aci-e2e/e2e-aci_test.go#L172.